code/_onclick/drag_drop.dm

```diff
@@ -7,10 +7,11 @@
 */
 /atom/MouseDrop(atom/over)
 	if(!usr || !over) return
-	if(!Adjacent(usr) || !over.Adjacent(usr)) return // should stop you from dragging through windows
+	if(!istype(over, /obj/screen))
+		if(!Adjacent(usr) || !over.Adjacent(usr)) return // should stop you from dragging through windows
```

Comment:
I know this was already like this but, can you put the return on its own line?

Reply:
Aye, done